tests/test_exist_client.py

```diff
@@ -89,3 +91,27 @@ def test_exist_client_retrieve_resources(test_client):
     retrieved_nodes_str = [str(node) for node in retrieved_nodes]
     assert node_resource in retrieved_nodes_str
     assert document_resource in retrieved_nodes_str
+
+
+@pytest.mark.usefixtures("db")
+@pytest.mark.parametrize(
+    "url, properties",
+    (
+        ("existdb://hostname/", ("https", "", "", "hostname", 443, "/")),
+        ("existdb+https://hostname/", ("https", "", "", "hostname", 443, "/")),
+        ("existdb+http://hostname/", ("https", "", "", "hostname", 80, "/")),
+        ("existdb+http://hostname:8080/", ("https", "", "", "hostname", 8080, "/")),
```

Comment:
I think the protocol for the properties is wrong here, should be `http`?

Reply:
indeed. maybe it won't be climate change but a c&p error that will initiate the coming apocalypse.